### PR TITLE
dynamic connection urls between home page and recipe page

### PR DESCRIPTION
**Pull request by jacobostapenko**

### CODE REVIEW

File: recipes/templates/recipes/base.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
         >
           <span class="navbar-toggler-icon"></span>
         </button>
-        <div class="collapse navbar-collapse" id="navbarNav">
+     <!--   <div class="collapse navbar-collapse" id="navbarNav">
```
Comment (dost4) [inline recipes/templates/recipes/base.html:33]: why are you commenting this out?

**Review by dost4 — APPROVED** [review]
This just hardcodes some text right? Add a query to pull the recipes out of the DB and display those next

File: recipes/templates/recipes/base.html (hunk 1 above)
Comment (jacobostapenko) [reply recipes/templates/recipes/base.html:33]: UI component that didn't make sense. planned to delete it

Comment (jacobostapenko) [reply recipes/templates/recipes/base.html:33]: it was a 'recipes home' button. assumed the big 'recipes' button would just bring to home page

**Pull request merged**